src/main/java/com/ecommerce/ECommerceApp/model/Order.java

```diff
@@ -1,5 +1,6 @@
 package com.ecommerce.ECommerceApp.model;
 
+import java.math.BigDecimal;
```

Comment:
Updated imports to include BigDecimal for precise financial calculations and enhanced data accuracy.